pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -66,6 +66,10 @@
   public static final String CONTROLLER_MODE = "controller.mode";
   public static final String LEAD_CONTROLLER_RESOURCE_REBALANCE_STRATEGY = "controller.resource.rebalance.strategy";
 
+  // Comma separated list of list of packages that contain TableConfigTuners to be added to the registry
+  public static final String TABLE_CONFIG_TUNER_PACKAGES = "controller.table.config.tuner.packages";
+  public static final String DEFAULT_TABLE_CONFIG_TUNER_PACKAGES = TableConfigTunerRegistry.class.getPackage().getName();
```

Comment:
This is `org.apache.pinot.common.config.tunner`, which is not the same as the current package `org.apache.pinot`

Reply:
Yes, I didn't want to hardcode it, and I also thought it might be better to be specific. But on second thought, better to keep previous behavior.